lang/de.json

```diff
@@ -4,18 +4,18 @@
     "single-tab-output-header-start": "Start (s)",
     "single-tab-output-header-end": "Ende (s)",
     "single-tab-output-header-sci-name": "Wissenschaftlicher Name",
-    "single-tab-output-header-common-name": "Gebräuchlicher Name",
+    "single-tab-output-header-common-name": "Trivialname",
```

Comment:
Does that word really exist?

Reply:
yes (https://de.wikipedia.org/wiki/Trivialname)